standard/standard/annex-ats.adoc

```diff
@@ -1,1085 +1,1288 @@
-<s|Purpose|Verify that this requirement is satisfied
-<s|Method|Inspection
-<s|Reference: G-LM|/req/lm_general
-<s|Test Type|Conformance
-|===
-
+[conformance_test,id="rta3"]
+====
+[%metadata]
+name:: G-LM
+label:: /conf/lm_general
+description:: Implementations of concrete data conforming to this standard SHALL
+conform to all dependent or inherited classes, attributes, and associations,
+multiplicities, and data types in the Logical Model.
+test-purpose:: Verify that this requirement is satisfied
+test-method:: Inspection
+reference:: <<lm_confomance,/req/lm_general>>
+classification:: Test Type:Conformance
+====
 
 === Structural Data Unit (SDU) Conformance
 
-There are some universal requirements on values that appear in a concrete implementation using a specific encoding technology. For example, angles may be constrainted to fall within a range of values correponding to a circle. Because these are independent of encoding technology, they are specified here at a logical level. Tests of an implementation at the SDU level generally only be done by inspection.
+There are some universal requirements on values that appear in a concrete
+implementation using a specific encoding technology. For example, angles may be
+constrainted to fall within a range of values correponding to a circle. Because these
+are independent of encoding technology, they are specified here at a logical level.
+Tests of an implementation at the SDU level generally only be done by inspection.
 
 === Basic-YPR SDU Conformance Class
 
 [[conf_class_basic_ypr_sdu]]
-[width="100%",%header,cols="3,6"]
-|===
-<|*Conformance Class:* <|*Basic-YPR SDU* 
-
-<s|ID |<<conf_class_basic_ypr_sdu,/conf/class/basic_ypr/sdu>>
-<s|Target Type|SDU
-<s|Description|To confirm that a Basic-YPR GeoPose consists of an Outer Frame specified by an implicit WGS-84 CRS and an implicit EPSG 4461-CS (LTP-ENU) coordinate system and explicit parameters defining the tangent point and that the Inner Frame is a rotation-only transformation using Yaw, Pitch, and Roll angles.
-<s|Dependency| /conf/global
-<s|Test: B-YPR-SDU|/conf/basic_ypr/sdu 
-<s|Test: B-TP-Lon-SDU |/conf/tangent_plane/longitude/sdu 
-<s|Test: B-TP-Lat-SDU |/conf/tangent_plane/latitude/sdu 
-<s|Test: B-TP-h-SDU |/conf/tangent_plane/h/sdu 
-<s|Test: YPR-Angles |/conf/ypr_angles/sdu 
-|===
+[conformance_class]
+====
+[%metadata]
+name:: Basic-YPR SDU
+label:: /conf/class/basic_ypr/sdu
+subject:: <<req_class_basic_ypr_sdu,Requirements Class "Basic YPR SDU">>
+classification:: Target Type:SDU
+description:: To confirm that a Basic-YPR GeoPose consists of an Outer Frame
+specified by an implicit WGS-84 CRS and an implicit EPSG 4461-CS (LTP-ENU) coordinate
+system and explicit parameters defining the tangent point and that the Inner Frame is
+a rotation-only transformation using Yaw, Pitch, and Roll angles.
+inherit:: /conf/global
+
+[conformance_test,label="/conf/basic_ypr/sdu",name="B-YPR-SDU",id="rta4"]
```

Comment:
These conformance tests are defined later. We should not re-define them here. This block should just be:
```adoc
conformance-test:: /conf/basic_ypr/sdu
```

Please change all the others too.